invenio_rdm_records/services/permissions.py

```diff
@@ -88,7 +89,9 @@ class RDMRecordPermissionPolicy(RecordPermissionPolicy):
     can_get_content_files = [
         # note: even though this is closer to business logic than permissions,
         # it was simpler and less coupling to implement this as permission check
-        IfFileIsLocal(then_=can_read_files, else_=[SystemProcess()])
+        IfFileIsLocal(
+            then_=can_read_files + [ResourceAccessToken()], else_=[SystemProcess()]
```

Comment:
Note: added RAT permission only to the `then_` of `IfFileIsLocal`, as in other case the resource is available only for `SystemProcess`

Question: should we allow reading the resource with a RAT even for non-local files?

Reply:
I think is okay, at the moment I don't see more use cases for `else_` than the fetch files, which will become "local" once downloaded.